specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -37,3 +37,11 @@ options:
     package-dir: "azure-contoso-widgetmanager"
     namespace: com.azure.contoso.widgetmanager
     flavor: azure
+  "@azure-tools/typespec-go":
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    service-dir: "sdk/contosowidget"
+    package-dir: "azmanager"
+    module-version: "0.0.1"
+    inject-spans: true
+    single-client: true
+    slice-elements-byval: true
```

Comment:
just mix all the config i noticed from current prs. need to confirm.

Reply:
`generate-fakes` should also be set to `true`.
`slice-elements-byval` should be `true` for greenfield only.
`single-client` should be `true` by default but may be `false` if the SDK is intended to contain multiple clients.

Reply:
Is `single-client` applicable for arm too?